source/Halibut/Transport/Streams/NetworkTimeoutStream.cs

```diff
@@ -2,10 +2,9 @@
 using System;
 using System.IO;
 using System.Net.Sockets;
-using System.Runtime.Remoting;
+using System.Runtime.Remoting; // Used in NETFRAMEWORK pragma
```

Comment:
If that // needed?

Reply:
Needed? No. I thought it might be helpful as a reminder in the future, as I cleared out this 'unused' `using` statement, and then wondered why it wouldn't compile.

Maybe I'll just be more careful next time I see pre-processor directives, and then I won't need this! 😄